contentcuration/contentcuration/frontend/shared/vuex/session/index.js

```diff
@@ -109,10 +103,9 @@ export default {
     login(context, credentials) {
       return client.post(window.Urls.login(), credentials);
     },
-    logout(context) {
-      return client.get(window.Urls.logout()).then(() => {
-        context.commit('REMOVE_SESSION');
-      });
+    logout() {
+      resetDB();
+      return client.get(window.Urls.logout());
```

Comment:
Maybe this should be:
```
return client.get(window.Urls.logout()).then(resetDB);
```

As, in combination with the delete listener, we probably want to make sure that the logout occurs server side before we clear the DB and initiate the redirect?